### PR TITLE
[HotFix] fix unpackdb issue in cmakefile

### PR DESCRIPTION
**Pull request by junliume**
after #2800

fix the issue:
```
CMake Error at CMakeLists.txt:566 (unpack_db):
  unpack_db Function invoked with incorrect arguments for function named:
  unpack_db
Call Stack (most recent call first):
  CMakeLists.txt:585 (install_kdb)

```

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -563,7 +563,7 @@ function(install_kdb FILE_NAME COMPONENT_NAME)
     if(__contents MATCHES "version")
         list(APPEND LFS_MISSING_FILES ${__fname})
     else()
-        unpack_db(${FILE_NAME})
+        unpack_db(${FILE_NAME} FILE_SUFFIX)
```
Comment (atamazov) [inline CMakeLists.txt:566]: [Recommendation]
```suggestion
        unpack_db(${FILE_NAME} UNUSED_FILE_SUFFIX)
```

Comment (atamazov) [reply CMakeLists.txt:566]: The design of this stuff is a bit clumsy, so I would also recommend adding a comment after line 532:
```
if(NOT MIOPEN_USE_SQLITE_PERFDB AND __extension STREQUAL ".db")
```
that should inform the reader that this branch is dedicated to a special case of unpacking ~system-find-db~ **_system-perf-db_** files converted to text format.

/cc @DrizztDoUrden

**Review by atamazov — APPROVED** [review]
LGTM albeit some recommendations

File: CMakeLists.txt (hunk 1 above)
Comment (atamazov) [reply CMakeLists.txt:566]: UPD: system-perf-db files

Comment (junliume) [reply CMakeLists.txt:566]: @atamazov should we close this PR and use #2802 instead? That's my plan

**Pull request closed without merging**